person.rb

```diff
@@ -0,0 +1,18 @@
+# Person class
+class Person
+  def initialize(age, name = 'Unknown', _parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+  end
+
+  attr_reader :id, :name, :age
```

Comment:
- [ ] It would be better if we set the getter and setter for our `name` and `age`. Since we are not only reading the values but also writing